i18n/en.pot

```diff
@@ -5,8 +5,8 @@ msgstr ""
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1)\n"
-"POT-Creation-Date: 2019-07-16T15:14:24.210Z\n"
-"PO-Revision-Date: 2019-07-16T15:14:24.210Z\n"
+"POT-Creation-Date: 2019-07-18T20:18:16.329Z\n"
+"PO-Revision-Date: 2019-07-18T20:18:16.329Z\n"
```

Comment:
Is this file up-to-date? When I run `yarn extract-pot`, the file has some changes. (for example, it adds literal "Deleting Notifications").